test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -110,4 +118,66 @@ describe('Your PSP Stripe page', () => {
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
     cy.get('span').contains('Government entity document').should('have.attr', 'href', `/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}/government-entity-document`)
   })
+
+  describe('Bank details task', () => {
+    it('should click bank details task and display bank details page correctly', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
+      cy.get('span').contains('Bank Details').click()
+      cy.get('#bank-details-form').should('exist')
```

Comment:
I would say we should just check for the H1.
To keep the test light - as the bank details cypress test already tests this.